src/time_domain_reduction/full_time_series_reconstruction.jl

```diff
@@ -0,0 +1,75 @@
+@doc raw"""full_time_series_reconstruction(path::AbstractString,
+                            case::AbstractString,
+                            DF::DataFrame,
+                            DFnames::Vector)
+Create a DataFrame with all 8,760 hours of the year from the reduced output.
+
+path - Path input to the results folder
+setup - case setup (dictionary)
+DF - DataFrame to be reconstructed
+DFnames - Vector of column names
+
+This function uses Period_map.csv to create a new DataFrame with 8,760 time steps, as well as other pre-existing rows such as "Zone".
+For each 52 weeks of the year, the corresponding representative week is taken from the input DataFrame and copied into the new DataFrame. Representative periods that 
+represent more than one week will appear multiple times in the output. 
+
+Note: Currently, TDR only gives the representative periods in Period_map for 52 weeks, when a (non-leap) year is 52 weeks + 24 hours. This function takes the last 24 hours of 
+the time series and copies them to get up to all 8,760 hours in a year.
+
+This function is called when output files with time series data (e.g. power.csv, emissions.csv) are created, if the setup key "OutputFullTimeSeries" is set to "1".
+
+"""
+
```

Comment:
The docstring needs to be attached to the function body for the Documenter to render it.
```suggestion
```